CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 All notable changes to this project will be documented in this file. This
 project adheres to [Semantic Versioning](http://semver.org/).
 
+## 0.50.2
+
+This is a bug fix release that addresses a regression in 0.50.1. 
```

Comment:
Perhaps we could include a simple example? It'll be easier to grasp what this is about, I think.

Reply:
Yep, I'll add one 👍 .